argument.go

```diff
@@ -21,12 +21,31 @@ type arg struct {
 	parent   *Command    // Used to get access to specific Command
 }
 
+// Arg interface provides exporting of arg structure, while exposing it
+type Arg interface {
+	GetOpts() *Options
+	GetSname() string
+	GetLname() string
+}
+
+func (o arg) GetOpts() *Options {
+	return o.opts
+}
+
+func (o arg) GetSname() string {
+	return o.sname
+}
+
+func (o arg) GetLname() string {
+	return o.lname
+}
+
 type help struct{}
 
 func (o *arg) check(argument string) bool {
 	// Shortcut to showing help
 	if argument == "-h" || argument == "--help" {
-		helpText := o.parent.Usage(nil)
+		helpText := o.parent.Help(nil)
```

Comment:
Is there a reason to rename it from `Usage()` to `Help()`? If not reason, then should not rename.

Reply:
Not a rename, Usage is still a function to avoid breaking changes. This is to enable help function overriding when using the -h | --help CLI arguments. Instead of calling Usage, Help is called, which defaults to Usage until overridden.

Reply:
Oh, right, silly me, was just scanning it quickly before. That's fine then.